src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -34,12 +35,42 @@
  * Calculate read-counts across targets given their reference coordinates.
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
+ *     You may obtain better results with different parameters.
```

Comment:
Modified top to "Calculates read-counts across targets for the exome copy number variant (CNV) calling workflow."